pages/home.tsx

```diff
@@ -0,0 +1,10 @@
+import React from 'react'
+import Menu from '../components/Menu'
+
+export default function home() {
```

Comment:
Welcome to the world Home page!

![](https://media.giphy.com/media/J93sVmfYBtsRi/giphy.gif)

Reply:
:O gee, I forgot to delete this one haha. I was using this to develop the menu XD (do I leave it there until we move it to"/" ?)

Reply:
yeah lets leave it there :)